pilot/cmd/pilot-agent/main.go

```diff
@@ -317,6 +330,13 @@ var (
 					if disableInternalTelemetry {
 						opts["DisableReportCalls"] = "true"
 					}
+
+					if sdsEnabled {
+						opts["sds_enabled"] = "true"
```

Comment:
Not sure we need sds_enabed. 
If sds_uds_path is "" -> sds will not be enabled by Pilot ( no matter what sds_enabled says ).
If it is a file - pilot will generate sds config. No need for sds_enabled.

Reply:
removed this for now (will add back if bootstrap json file doesn't support ```if ne``` semantic later)